deep_qa/data/instances/sentence_selection_instance.py

```diff
@@ -0,0 +1,161 @@
+from typing import Dict, List
+
+import numpy as np
+from overrides import overrides
+
+from .instance import TextInstance, IndexedInstance
+from ..data_indexer import DataIndexer
+
+
+class SentenceSelectionInstance(TextInstance):
+    """
+    A SentenceSelectionInstance an instance for the sentence selection
+    task. A SentenceSelectionInstance stores a question as a string, and a set of sentences
+    as a list of strings. The labels is a single int, indicating the index of
+    the sentence that contains the answer to the question.
+    """
+    def __init__(self, question_text: str, sentences: List[str], label: int, index: int=None):
+        super(SentenceSelectionInstance, self).__init__(label, index)
+        self.question_text = question_text
+        self.sentences = sentences
+
+    def __str__(self):
+        return ('SentenceSelectionInstance(' + self.question_text +
+                ', ' + self.sentences + ', ' +
+                str(self.label) + ')')
+
+    @overrides
+    def words(self) -> Dict[str, List[str]]:
+        words = self._words_from_text(self.question_text)
+        sentences_words = [self._words_from_text(sentence) for sentence in
+                           self.sentences]
+        for namespace in words:
+            for sentence_words in sentences_words:
+                words[namespace].extend(sentence_words[namespace])
+        return words
+
+    @overrides
+    def to_indexed_instance(self, data_indexer: DataIndexer):
+        question_indices = self._index_text(self.question_text, data_indexer)
+        sentences_indices = [self._index_text(sentence, data_indexer) for
```

Comment:
I think it's more readable to have the `for` on the next line.  Do you think this way is better?